src/libServer/LookupServer.cpp

```diff
@@ -1159,6 +1185,93 @@ Json::Value LookupServer::GetEthBlockTransactionCountByNumber(
   }
 }
 
+Json::Value LookupServer::GetEthTransactionByBlockHashAndIndex(
+    const std::string& inputHash, const std::string& indexStr) const {
+  try {
+    const BlockHash blockHash{inputHash};
+    const auto txBlock = m_mediator.m_txBlockChain.GetBlockByHash(blockHash);
+    const uint64_t index = std::strtoull(indexStr.c_str(), nullptr, 0);
```

Comment:
This looks like it will not work as the index will likely be prepended with `0x` https://www.quicknode.com/docs/ethereum/eth_getTransactionByBlockHashAndIndex

Reply:
Thanks for raising that, however it seems it's going to work as expected. According to docs (https://en.cppreference.com/w/cpp/string/byte/strtoul):
Signature is:
`unsigned long long strtoull( const char *str, char **str_end, int base );`
And a comment on base (last parameter):
`If the value of base is ​0​, the numeric base is auto-detected: if the prefix is 0, the base is octal, if the prefix is 0x or 0X, the base is hexadecimal, otherwise the base is decimal.`